gapis/api/gvr/.gitignore

```diff
@@ -1 +1,2 @@
 /api.pb.go
+resolvables.pb.go
```

Comment:
Add the slash in the front, it means to only exclude it in the current dir rather than in it and it's children.

Reply:
done